examples/crowdfunding/src/lib.rs

```diff
@@ -0,0 +1,157 @@
+
+#![no_std]
+#![allow(unused_attributes)]
+
+imports!();
+
+#[derive(PartialEq, Clone, Copy)]
+pub enum Status 
+{
+    FundingPeriod,
+    Successful,
+    Failed
+}
+
+#[elrond_wasm_derive::contract(CrowdfundingImpl)]
+pub trait Crowdfunding
+{
+    #[storage_set("owner")]
+    fn set_owner(&self, address: &Address);
+
+    #[view]
+    #[storage_get("owner")]
+    fn get_owner(&self) -> Address;
+
+    #[storage_set("target")]
+    fn set_target(&self, target: &BigUint);
+
+    #[view]
+    #[storage_get("target")]
+    fn get_target(&self) -> BigUint;
+
+    #[storage_set("deadline")]
+    fn set_deadline(&self, deadline: u64);
+
+    #[view]
+    #[storage_get("deadline")]
+    fn get_deadline(&self) -> u64;
+
+    #[storage_set("deposit")]
+    fn set_deposit(&self, donor: &Address, amount: &BigUint);
+
+    #[view]
+    #[storage_get("deposit")]
+    fn get_deposit(&self, donor: &Address) -> BigUint;
+
+    #[init]
+    fn init(&self, target: &BigUint, deadline: u64)
+    {
```

Comment:
The Rust standard seems to be with braces on the same line (most of the time).

Reply:
https://doc.rust-lang.org/1.0.0/style/style/braces.html

Reply:
So let's stick to the standards

Reply:
I sometimes leave the brace in the next line, but only when there is a trait or function with a lot of "where" clauses. Very very seldom